### PR TITLE
show long code expiry in UI on code check

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* Long code expiration displayed in code check UI

image

**Release Note**

```release-note
When checking code status via UUID, the long expiry time/countdown will be shown.
```

### CODE REVIEW

File: cmd/server/assets/codestatus/show.html [hunk 1]
```diff
@@ -42,6 +42,12 @@ <h5 class="mb-1">Status</h5>
         <h5 class="mb-1">Expiry</h5>
         <span id="code-expires-at" class="sm text-danger"></span>
       </div>
+      {{if .code.HasLongExpires}}
+      <div class="list-group-item">
+        <h5 class="mb-1">Long code expiry</h5>
```
Comment (sethvargo) [inline cmd/server/assets/codestatus/show.html:47]: I don't think we call this "long code" anywhere but the database. Should we say "SMS verification link" instead?

File: cmd/server/assets/codestatus/show.html [hunk 2]
```diff
@@ -67,6 +74,15 @@ <h5 class="mb-1">Expiry</h5>
         // Disable the submit if already expired.
         $buttonInvalidate.prop('disabled', true);
       });
+
+      {{if .code.HasLongExpires}}
+      let $longCodeExpiresAt = $('#long-code-expires-at');
+      // Start countdown
+      countdown($longCodeExpiresAt, longExpires, function() {
```
Comment (sethvargo) [inline cmd/server/assets/codestatus/show.html:81]: do-da-do---do, da-da-do-do-do, do-da-do---do...

File: cmd/server/assets/codestatus/show.html (hunk 1 above)
Comment (mikehelmick) [reply cmd/server/assets/codestatus/show.html]: changed to "SMS link expiry"

File: cmd/server/assets/codestatus/show.html (hunk 2 above)
Comment (mikehelmick) [reply cmd/server/assets/codestatus/show.html:82]: you ok?

also - I made this better

Comment (sethvargo) [reply cmd/server/assets/codestatus/show.html:82]: It's really hard to put musical intonation into words. I was going for: https://www.youtube.com/watch?v=9jK-NcRmVcw

Comment (mikehelmick) [reply cmd/server/assets/codestatus/show.html:82]: https://www.youtube.com/watch?v=bagCdMUuevQ

**Pull request merged**